rapids_dask_dependency/dask_loader.py

```diff
@@ -57,19 +59,19 @@ def disable(self, name):
     def find_spec(self, fullname: str, _, __=None):
         if fullname in self._blocklist:
             return None
-        if (
-            fullname in ("dask", "distributed")
-            or fullname.startswith("dask.")
-            or fullname.startswith("distributed.")
-        ):
-            return importlib.machinery.ModuleSpec(
+        if fullname.startswith("dask") or fullname.startswith("distributed"):
```

Comment:
Do we need to worry about the hypothetical case that there is a library called something like `daskly`?

Reply:
Great call. I got caught up in all the simplifying I was doing in this PR and forgot that there was a good reason for this. Reverted.